ocaml/typing/jkind_types.mli

```diff
@@ -54,21 +54,35 @@ module Sort : sig
     | Bits32
     | Bits64
 
+  val to_string_base : base -> string
```

Comment:
Since `Sort.t` now depends on `base`, attempts to move it completely into its own module run into the same issue previously encountered with `const`, described in response to the previous question.  And I think half moving it is more confusing than not, so I've just left it.